Sources/MapboxCoreNavigation/TilesetDescriptorFactory.swift

```diff
@@ -14,14 +14,40 @@ extension TilesetDescriptorFactory {
     }
 
     /**
-     Gets TilesetDescriptor which corresponds to the latest availble version of routing tiles.
-     
-     Intended for using when creating off-line tile packs.
-     
-     - parameter location: A `Location` where cache data is stored.
+     Gets TilesetDescriptor that corresponds to the latest available version of routing tiles.
+
+     It is intended to be used when creating off-line tile packs.
+
+     - Parameters:
+       - location: A `Location` where cache data is stored.
+       - completionQueue: A DispatchQueue on which the completion will be called.
+       - completion: A completion that will be used to pass the latest tileset descriptor.
+       - latestTilesetDescriptor: A tileset descriptor for the latest available version of routing tiles.
```

Comment:
This was added so that it will show up in the Xcode documentation popup, but it turns out it is very buggy and doesn't work as expected. Removing this.